gammapy/modeling/models/spatial.py

```diff
@@ -622,7 +624,7 @@ class GaussianSpatialModel(SpatialModel):
     lat_0 = Parameter("lat_0", "0 deg", min=-90, max=90)
     sigma = Parameter("sigma", "1 deg", min=0)
     e = Parameter("e", 0, min=0, max=1, frozen=True)
-    phi = Parameter("phi", "0 deg", frozen=True)
+    phi = Parameter("phi", "0 deg", frozen=True, min=-20, max=200)
```

Comment:
does this limit the fitted error if the true error is larger than 20 ?

Reply:
Thanks @QRemy ! Actually I could not simulate a source with significant ellipticity, but error on angle greater than 20.
But you are right, 20 is a bad number.
I now put it between -180 and 180.